src/components/TextField/TextField.st.css

```diff
@@ -165,13 +170,19 @@
     background: applyOpacity(value(mainTextColor), 0.2);
 }
 
+.root:theme(line).suffixWrapper {
+    padding: 0;
+}
+
 .suffixWrapper {
     height: 24px;
     display: flex;
     margin: 0 -4px;
     justify-content: flex-end;
     white-space: nowrap;
     vertical-align: middle;
+    padding-left: 12px;
+    padding-right: 12px;
```

Comment:
It seems that this is working for the SuccessIcon suffix, but not the ErrorIcon

Reply:
are you sure? because its the same component with same class just different Icon inside